src/components/BasicTabs.js

```diff
@@ -9,13 +9,12 @@ import Tabs, { Tab } from "material-ui/Tabs"
 import Courses from "./Courses"
 import Grades from "./Grades"
 import { translate, Interpolate } from "react-i18next"
-import i18n from "./utils/i18n"
+import i18n from "./../utils/i18n"
 
-const TabContainer = props => (
+const TabContainer = props =>
```

Comment:
Wow, this is an interesting syntax. I would have assumed the parentheses were necessary